site/_sass/styles.scss

```diff
@@ -18,8 +18,9 @@
 @forward "usa-skipnav";
 @forward "usa-media-block";
 @forward "usa-prose";
+@forward "usa-prose";
```

Comment:
Looks like we have a duplicate forward of `usa-prose` on line 20.

```suggestion
```

Reply:
I'm going top leave this as-is to avoid any more changes in this release, but we'll add it to the next one

Reply:
Sounds good, captured in #334.